hwbench/environment/block_devices.py

```diff
@@ -69,6 +75,15 @@ def get_udev_attributes(self) -> dict[str, Any]:
             )
         return dumped
 
+    def get_manufacturer(self) -> str:
+        if not self.wwn:
+            return "000000"
```

Comment:
Why returning 00000 while we expect an manufacturer name ?

Reply:
fixed